app/src/main/res/values/strings.xml

```diff
@@ -35,6 +35,7 @@
     <string name="navigation_drawer_open">Open navigation drawer</string>
     <string name="navigation_drawer_close">Close navigation drawer</string>
     <string name="please_click_back_again_to_exit">Please click BACK again to exit!</string>
+    <string name="login_success">Login success</string>
```

Comment:
WARNING: [UnusedResources] The resource `R.string.login_success` appears to be unused
 Unused resources make applications larger and slow down builds.